### PR TITLE
TEVA-3446 Add an interstitial page with a button for confirming email address

### PR DESCRIPTION
**Pull request by elliotcm**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-3446

## Screenshots of UI changes:

Screenshot 2022-01-07 at 15 38 55

### CODE REVIEW

File: spec/system/jobseekers_can_register_spec.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
     click_on I18n.t("jobseekers.registrations.check_your_email.resend_link"), visible: false
     visit first_link_from_last_mail
+    click_on "Confirm"
```
Comment (josephhull676) [inline spec/system/jobseekers_can_register_spec.rb:18]: Would you `confirm_email_address` here too?

**Review by josephhull676 — APPROVED** [review]

File: spec/system/jobseekers_can_register_spec.rb (hunk 1 above)
Comment (elliotcm) [reply spec/system/jobseekers_can_register_spec.rb:18]: While they do the same thing, I wanted to be more explicit about the two steps here as this is what's actually under test, whereas in the other cases it's a step on the way to testing something else.

File: spec/system/jobseekers_can_sign_up_to_an_account_spec.rb [hunk 2]
```diff
@@ -37,41 +37,13 @@
       context "when the confirmation period has expired" do
         before { travel_to 3.hours.from_now }
 
-        it "does not confirm email and redirects to resend confirmation page" do
```
Comment (josephhull676) [inline spec/system/jobseekers_can_sign_up_to_an_account_spec.rb:40]: Why are you deleting these tests?

Comment (elliotcm) [reply spec/system/jobseekers_can_sign_up_to_an_account_spec.rb:40]: The logging was put in place by David when it was under investigation.  I removed that code so these events are no longer fired.

**Pull request merged**